src/Umbraco.Web/Editors/BackOfficeServerVariables.cs

```diff
@@ -328,6 +331,14 @@ public BackOfficeServerVariables(UrlHelper urlHelper, ApplicationContext applica
                                 .ToArray()
                         }
                     }
+                },
+                {
+                    "disabledFeatures", new Dictionary<string,bool>
```

Comment:
We don't a dictionary of string,bool, this will not be very flexible if a new feature isn't a bool, leaving it as `object` is fine